portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,47 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Antonio!</h1>");
+    ArrayList<String> data = createArrayList();
+    String json = convertToJson(data);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Creates an ArrayList with example data.
+   * @return an ArrayList of strings with hardcoded data.
+   */
+  public static ArrayList<String> createArrayList() {
+    ArrayList<String> exampleData = new ArrayList<String>();
+
+    exampleData.add("Apple");
```

Comment:
You could use `new ArrayList<String>(Arrays.asList(...))` to perfrom this entire operation in a single line.

Reply:
I played around with this syntax for a bit but couldn't find a way to do it without erroring. Thank you! Since this method gets deleted in the next PR I may just decide to leave it as-is.